docs/elasticsearch/ux.md

```diff
@@ -0,0 +1,1081 @@
+---
+layout: default
+title: Search Experience
+parent: Elasticsearch
+nav_order: 12
+---
+
+# Search Experience
+
+Expectations from search engines have evolved over the years. Just returning relevant results quickly is no longer enough for most users. Elasticsearch includes many features that enhance the user’s search experience as follows:
+
+Feature | Description
+:--- | :---
+Autocomplete queries | Suggest phrases as the user types.
+Paginate results |  Rather than a single, long list, break search results into pages.
+Scroll search | Return a large number of results in batches.
+Sort results | Allow sorting results by different criteria.
+Highlight query matches | Highlight the search term in the results.
+
+---
+
+## Autocomplete queries
+
+Autocomplete shows suggestions to users while they type.
+
+For example, if a user types "pop," Elasticsearch provides suggestions like "popcorn" or "popsicles." These suggestions preempt your user's intention and lead them to a possible search term more quickly.
+
+Elasticsearch allows you to design autocomplete that’s:
+
+- Responsive: Updates suggestions with each keystroke.
+- Relevant: Serves a few but relevant suggestions.
+- Forgiving: Tolerates typos.
+
+Implement autocomplete in one of three methods:
+
+- Prefix matching
+- Edge N-gram matching
+- Completion suggesters
+
+These methods are described below.
```

Comment:
Can we make a recommendation here, or it's just "each one has pros and cons?"

Reply:
There isn't clear winner, it depends on the use case. We could have a future section with detailed pros and cons for each method.